daprdocs/content/en/reference/components-reference/supported-state-stores/setup-cloudflare-workerskv.md

```diff
@@ -0,0 +1,242 @@
+| `workerName` | Y | Name of the Worker to connect to | `"mydaprkv"`
+| `key` | Y | Ed25519 private key, PEM-encoded | *See example above*
+| `cfAccountID` | Y/N | Cloudflare account ID. Required to have Dapr manage the worker. | `"456789abcdef8b5588f3d134f74ac"def`
+| `cfAPIToken` | Y/N | API token for Cloudflare. Required to have Dapr manage the Worker. | `"secret-key"`
+| `workerUrl` | Y/N | URL of the Worker. Required if the Worker has been pre-provisioned outside of Dapr. | `"https://mydaprkv.mydomain.workers.dev"`
+
+> When you configure Dapr to create your Worker for you, you may need to set a longer value for the `initTimeout` property of the component, to allow enough time for the Worker script to be deployed. For example: `initTimeout: "120s"`
+
+## Create a Workers KV namespace
+
+To use this component, you must have a Workers KV namespace created in your Cloudflare account.
+
+You can create a new Workers KV namespace in one of two ways:
+
+<!-- IGNORE_LINKS -->
+- Using the [Cloudflare dashboard](https://dash.cloudflare.com/)  
+  Make note of the "ID" of the Workers KV namespace that you can see in the dashboard. This is a hex string (for example `123456789abcdef8b5588f3d134f74ac`)–not the name you used when you created it!
+- Using the [Wrangler CLI](https://developers.cloudflare.com/workers/wrangler/):
+  
+  ```sh
+  # Authenticate if needed with `npx wrangler login` first
+  wrangler kv:namespace create <NAME>
+  ```
+
+  The output contains the ID of the namespace, for example:
+
+  ```text
+  { binding = "<NAME>", id = "123456789abcdef8b5588f3d134f74ac" }
+  ```
+<!-- END_IGNORE -->
+
+## Configuring the Worker
+
+Because Cloudflare Workers KV namespaces can only be accessed by scripts running on Workers, Dapr needs to maintain a Worker to communicate with the Workers KV storage.
+
+Dapr can manage the Worker for you automatically, or you can pre-provision a Worker yourself. Pre-provisioning the Worker is the only supported option when running on [workerd](https://github.com/cloudflare/workerd).
+
+> **Important:** Use a separate Worker for each Dapr component. Do not use the same Worker script for different Cloudflare Workers KV state store components, and do not use the same Worker script for different Cloudflare components in Dapr (e.g. the Workers KV state store and the Queues binding).
+
+{{< tabs "Let Dapr manage the Worker" "Manually provision the Worker script" >}}
+
+{{% codetab %}}
+<!-- Let Dapr manage the Worker -->
+
+If you want to let Dapr manage the Worker for you, you will need to provide these 3 metadata options:
+
+<!-- IGNORE_LINKS -->
+- **`workerName`**: Name of the Worker script. This will be the first part of the URL of your Worker: for example, if the "workers.dev" domain configured for your Cloudflare account is `mydomain.workers.dev` and you set `workerName` to `mydaprkv`, the Worker that Dapr deploys will be available at `https://mydaprkv.mydomain.workers.dev`.
```

Comment:
```suggestion
- **`workerName`**: Name of the Worker script. This will be the first part of the URL of your Worker. For example, if the "workers.dev" domain configured for your Cloudflare account is `mydomain.workers.dev` and you set `workerName` to `mydaprkv`, the Worker that Dapr deploys will be available at `https://mydaprkv.mydomain.workers.dev`.
```